### ISSUE TITLE
Melted SD Card

### ISSUE THREAD

**Issue by anaknaga** [voidlinux/void-packages#7009]
So, I dd'ed VoidLinux to my USB disk, rebooted my laptop, selected boot (copy to ram), and went to make a coffee. When I came back I noticed that the installer was still booting, and doing something with mmcblk0 (which I know is my microsd card, in my sd card adapter, in my thinkpad t450s slot). So I just took it out, while the installer was still busy, and nearly burned my fingers! The plastic around the microsd card, inside the sd card adapter, is slightly melted. I don't know in what state the microsd card is, as it doesn't come out anymore, but I guess I'm not using that one anymore!

So please update the wiki to include a fire warning for microsd cards. Thanks!

**Comment by chneukirchen** [comment]
Please retry with a different distribution, to see if this is a Void-specific issue.

**Comment by anaknaga** [comment]
Well I'm not going to do that, but I tried many linux distributions over the years. Honestly it could be a faulty microsd card on my side, I just found it in an old tablet of my grandparents and didn't check it yet. 

It said something about buffering before I unplugged it, but I don't remember what exactly. Feel free to close this if you feel there's not enough to go on. Just thought I should share this.
